[CUBRIDMAN-175] A new system parameter oracle_compat_number_behavior (#439)
CUBRID/cubrid-manual: en/admin/config.rst

        ERROR: Attributes exposed in aggregate queries must also appear in the group by clause.

.. _oracle_compat_number_behavior:

**oracle_compat_number_behavior**

>     **oracle_style_empty_string** is a parameter used to improve compatibility with other DBMS (Database Management Systems); for NUMERIC type, DOUBLE and FLOAT types, it does not display decimal point trailing 0, and in case of DOUBLE and FLOAT, it does not display the exponent. For example, if this parameter setting value is **no**, the result of a query that searches the a_double table composed of DOUBLE types is displayed in the form of an exponent as shown below, but if the parameter setting value is **yes**, only decimal point is displayed.

Reviewer: oracle_style_empty_string
-> oracle_compat_number_behavior

Author: Ok, good point. Thanks. I fixed.

>     **oracle_style_empty_string** is a parameter used to improve compatibility with other DBMS (Database Management Systems); for NUMERIC type, DOUBLE, and FLOAT types, it does not display decimal point trailing 0, and in case of DOUBLE and FLOAT, it does not display the exponent. For example, if this parameter setting value is **no**, the result of a query that searches the a_double table composed of double types is displayed in the form of an exponent as shown below, but if the parameter setting value is **yes**, only decimal point is displayed.

Reviewer 2: ```suggestion
    **oracle_style_empty_string** is a parameter used to improve compatibility with other DBMS (Database Management Systems); for NUMERIC type, DOUBLE and FLOAT types, it does not display decimal point trailing 0, and in case of DOUBLE and FLOAT, it does not display the exponent. For example, if this parameter setting value is **no**, the result of a query that searches the a_double table composed of DOUBLE types is displayed in the form of an exponent as shown below, but if the parameter setting value is **yes**, only decimal point is displayed.
```

    .. code-block:: sql

>         csql> ;Get oracle_compat_number_behavior

Reviewer 2: ```suggestion
        csql> ;get oracle_compat_number_behavior
```

        === Get Param Input ===

        oracle_compat_number_behavior=n

        SELECT a FROM a_double;

    ::

                                 a
        ==========================
             1.234567890123457e+19
             1.234567890123457e-08
             1.230000000000000e-08
             1.000000000000000e+00
             1.200000000000000e+00
            -4.939030300000000e-03
            -1.293934894993939e+18
            -1.938943893939394e+16

    ::

>         csql> ;Get oracle_compat_number_behavior

Reviewer 2: ```suggestion
        csql> ;get oracle_compat_number_behavior
```

        === Get Param Input ===

        oracle_compat_number_behavior=y

        SELECT a FROM a_double;

    ::

                                 a
        ==========================
              12345678901234570000
          0.00000001234567890123457
                      0.0000000123
                                 1
                               1.2
                     -0.0049390303
              -1293934894993939000
                -19389438939393940

>     Also, when **oracle_compat_number_behavior** is set to **yes**, the type of division operation for integer numbers becomes a real number type, that is, a NUMERIC type, not an integer type. In the example below, if this parameter is **yes**, applicable integer types are all INT, SHORT, and BIGINT.

Reviewer 2: ```suggestion
    Also, when **oracle_compat_number_behavior** is set to **yes**, the type of division operation for integer numbers becomes a real number type (that is, a NUMERIC type) not an integer type. In the example below, if this parameter is **yes**, affected integer types are INT, SHORT and BIGINT.
```

    .. code-block:: sql

>         csql> ;Get oracle_compat_number_behavior

Reviewer 2: ```suggestion
        csql> ;get oracle_compat_number_behavior
```
